samples/highcharts/exporting/apply-stylesheets/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Highcharts Demo
+ authors:
+   - Torstein Hønsi
```

Comment:
🤔

Reply:
I was unsure of this because it was your original demo, just slightly altered by me!! 😆